src/mono/wasm/runtime/dotnet.d.ts

```diff
@@ -163,43 +163,28 @@ declare type MonoConfigError = {
     message: string;
     error: any;
 };
-declare type AllAssetEntryTypes = AssetEntry | AssemblyEntry | SatelliteAssemblyEntry | VfsEntry | IcuData;
-declare type AssetEntry = {
+interface ResourceRequest {
+    name: string;
+    behavior: AssetBehaviours;
+    resolvedUrl?: string;
+    hash?: string;
+}
+interface AssetEntry extends ResourceRequest {
     name: string;
     behavior: AssetBehaviours;
     virtual_path?: string;
     culture?: string;
+    resolvedUrl?: string;
```

Comment:
Why do we need to specify these again? Don't we inherit them by extending ResourceRequest?